www/src/components/creative-layout.js

```diff
@@ -6,7 +6,9 @@ export default ({ children }) => {
   return <div className={styles.container}>
     <Header/>
     <main>
-      {children}
+      <section>
+        {children}
+      </section>
     </main>
     <footer><p>Copyright © 2020 Hidden Switch</p></footer>
```

Comment:
![localhost_8000_(iPhone X) (7)](https://user-images.githubusercontent.com/2229300/86048203-f4ec8c00-ba04-11ea-9e77-392be997da22.png)

right now the footer is weirdly fixed to the bottom of the screen if you scroll all the way down to it on index.js and then scroll back up